lib/matplotlib/figure.py

```diff
@@ -1264,13 +1264,16 @@ def add_subplot(self, *args, **kwargs):
 
         Parameters
         ----------
-        *args, int or (int, int, int) or `SubplotSpec`, default: (1, 1, 1)
+        *args, int, (int, int, *index*), or `SubplotSpec`, default: (1, 1, 1)
             The position of the subplot described by one of
 
             - Three integers (*nrows*, *ncols*, *index*). The subplot will
               take the *index* position on a grid with *nrows* rows and
               *ncols* columns. *index* starts at 1 in the upper left corner
-              and increases to the right.
+              and increases to the right. *index* can also be  a two-tuple
+              specifying the (*start*, *stop*) indices of the subplot, i.e.,
```

Comment:
Oops, just realized this is an e.g., not i.e..
```suggestion
              specifying the (*start*, *stop*) indices of the subplot, e.g.,
```